vignettes/vroom.Rmd

```diff
@@ -112,6 +112,12 @@ file <- vroom_example("mtcars.csv.gz")
 vroom(file)
 ```
 
+About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).
```

Comment:
I think you may mean temporary where you're using temporal?

Reply:
Yes, thank you! :)